idseq_dag/steps/generate_taxid_fasta.py

```diff
@@ -33,7 +33,7 @@ def run(self):
 
         with open(input_fa_name, 'rb') as input_fa, \
              open(self.output_files_local()[0], 'wb') as output_fa, \
-             open_file_db_by_extension(lineage_db, IdSeqDictValue.VALUE_TYPE_ARRAY) as lineage_map:
+             open_file_db_by_extension(lineage_db, IdSeqDictValue.VALUE_TYPE_ARRAY) as lineage_map:  # noqa
```

Comment:
Why noqa here?

Reply:
Because of line break style rules that are too hard to fix without fully rewriting this statement.